docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.6"
+
+services:
+  marklogic:
+    image: marklogic.azurecr.io/marklogic:latest
+    volumes:
+    - type: volume
+      source: marklogic_data
+      target: /data
+    ports:
+    - 7997-8020:7997-8020/tcp
+    - 8855-8856:8855-8856/tcp
+  gradle:
```

Comment:
This is a cool gradle container trick.  When you run docker-compose it will start a gradle container and run some predefined gradle containers.  Allows more turnkey deployment.